demos/embeddings/stellargraph-keras-node2vec.ipynb

```diff
@@ -0,0 +1,464 @@
+{
```

Comment:
<p>Same as above, can remove one of these</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/536/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
I have deleted one of them.